src/index.js

```diff
@@ -0,0 +1,146 @@
+const downButton = document.getElementById('down-arrow');
+const tempText = document.getElementById('current-temp');
+const ground = document.getElementById('ground');
+const cityHeader = document.getElementById('city-name');
+const cityInput = document.getElementById('city');
+const realTemp = document.getElementById('realtime-temp');
+const skySelect = document.getElementById('sky-select');
+const skyLine = document.getElementById('sky');
+const landscape = document.getElementById('landscape');
+const cityReset = document.getElementById('city-reset');
+
+const setTempColor = (temp) => {
+  if (temp >= 80) {
+    tempText.style.color = 'red';
+  } else if (temp >= 70) {
+    tempText.style.color = 'orange';
+  } else if (temp >= 60) {
+    tempText.style.color = 'yellow';
+  } else if (temp >= 50) {
+    tempText.style.color = 'green';
+  } else {
+    tempText.style.color = 'teal';
+  }
+};
+
+const setGroundLayout = (temp) => {
+  if (temp >= 80) {
+    ground.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (temp >= 70) {
+    ground.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (temp >= 60) {
+    ground.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else {
+    ground.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+};
+
+const setSkyLayout = (sky) => {
+  if (sky === 'sunny') {
+    skyLine.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+    landscape.style.backgroundColor = '#76e0f5';
+  } else if (sky === 'cloudy') {
+    skyLine.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+    landscape.style.backgroundColor = '#8fc9f2';
+  } else if (sky === 'rainy') {
+    skyLine.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+    landscape.style.backgroundColor = '#4a73a8';
+  } else if (sky === 'snowy') {
+    skyLine.textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+    landscape.style.backgroundColor = '#a5c6c7';
+  }
+};
+
+const increaseTemp = () => {
+  state['temp'] += 1;
+  setTempColor(state['temp']);
+  tempText.textContent = state['temp'];
+  setGroundLayout(state['temp']);
+};
+
+const decreaseTemp = () => {
+  state['temp'] -= 1;
+  setTempColor(state['temp']);
+  tempText.textContent = state['temp'];
+  setGroundLayout(state['temp']);
+};
+
+const changeCityHeader = () => {
+  cityHeader.textContent = cityInput.value;
+};
+
+const getCityLatAndLon = () => {
+  state['city'] = cityInput.value;
+
+  axios
+    .get('http://localhost:5000/location', {
+      params: { q: state['city'] },
+    })
+    .then((response) => {
+      state['lat'] = response.data[0].lat;
+      state['lon'] = response.data[0].lon;
+    });
+};
+
+const changeToRealTemp = () => {
+  axios
+    .get('http://localhost:5000/weather', {
+      params: { lat: state['lat'], lon: state['lon'] },
+    })
+    .then((response) => {
+      const newTempKelvin = response.data.current.temp;
+      const newTempF = 1.8 * (newTempKelvin - 273) + 32;
+      state['temp'] = Math.round(newTempF);
+      setTempColor(state['temp']);
+      tempText.textContent = state['temp'];
+      setGroundLayout(state['temp']);
```

Comment:
You could put this logic from lines 101-105 in a helper method, maybe called `setRealTempLayout`, which would make the code in `then()` more concise.

Someone reading the then block, would see that with the response, you set the real temperature and corresponding layout.